src/test/java/org/assertj/core/api/offsetdatetime/OffsetDateTimeAssert_hasSameInstantAs_Test.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2019 the original author or authors.
+ */
+package org.assertj.core.api.offsetdatetime;
+
+import static java.lang.String.format;
+import static java.time.OffsetDateTime.of;
+import static org.assertj.core.api.AbstractOffsetDateTimeAssert.NULL_OFFSET_DATE_TIME_PARAMETER_MESSAGE;
+import static org.assertj.core.api.Assertions.*;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+
+import org.assertj.core.api.BaseTest;
+import org.junit.jupiter.api.Test;
+
+public class OffsetDateTimeAssert_hasSameInstantAs_Test extends BaseTest {
```

Comment:
nothing wrong with this class, it would be better to follow `ObjectArrays_assertHasSameSizeAs_with_Iterable_Test` ways, we found it to be a bit more readable ;-)

Reply:
I changed `OffsetDateTimeAssert_isAtSameInstantAs_Test` to extend from `OffsetDateTimeAssertBaseTest`, adding two new static `OffsetDateTimes`. Let me know if I should have done something differently.